src/api/verification_library.py

```diff
@@ -100,3 +111,79 @@ def get_library_item(self, item_name: str) -> Dict:
         }
 
         return item_dict
+
+    def validate_library(self, items: List[str] = []) -> pd.DataFrame:
+        """Check the validity of library items definition. This validity check includes checking the completeness of json specification (against library json schema) and Python verification class definition (against library class interface) and the match between the json and python implementation.
+
+        Args:
+            items: list of str, default []. Library items to validate. By default, summarize all library items loaded at instantiation.
+
+        Returns:
+            pandas.DataFrame that contains validity information of library items.
```

Comment:
Was there a discussion about the type of input? I was wondering if a `dict` could perhaps be simpler?

Reply:
Jerry and I briefly discussed this, but returning a ```dict``` seems like a good idea. @leijerry888 What do you think?

Reply:
Sorry, I meant output, in place of a `pd.DataFrame`.

Reply:
Got it. @lymereJ @yunjoonjung-PNNL Yeah, returning a dict is good considering we may want to further process that. Let's do that!